.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -16,9 +16,13 @@ assignees: ''
 
 <!-- Please add a screenshot if applicable -->
 
-[Optional] **Additional Context**
+
```

Comment:
I don't think we need to add more white space.

What do you think?

Reply:
I personally like it being a bit spaced out. Cramped headings can intimidate newcomers.

Reply:
We normally use 1 blank line to keep everything consistent

Reply:
Okay, fine. Will commit the change.